docs/cubeviz/plugins.rst

```diff
@@ -274,3 +274,23 @@ Export Plot
 ===========
 
 This plugin allows exporting the plot in a given viewer to various image formats.
+
+.. _cubeviz-export-video:
+
+Movie
+-----
+
+.. note::
+
+    For MPEG-4, this feature needs ``opencv-python`` to be installed;
+    see [opencv-python on PyPI](https://pypi.org/project/opencv-python/).
+
+Expand the "Export to video" section, then enter the desired starting and
+ending slice indices (inclusive), the frame rate in frames per second (FPS),
+and the filename.
+If a path is not given, the file will be saved to current working
+directory. Any existing file with the same name will be silently replaced.
+
+When you are ready, click the :guilabel:`Export to MP4` button.
+The movie will be recorded at 5 frames per second. While recording is in progress,
```

Comment:
Is the 5 FPS just a default or is it hard-coded? In the UI it looks like you can change it (vs the implication here that you can't change it), but I didn't try changing before writing out my first test movie.

Reply:
Ooops, doc is outdated. Good catch!

Reply:
Fixed.